app/assets/stylesheets/welcome.scss

```diff
@@ -85,21 +91,24 @@ iframe{
 .instagram-comment{
   padding-top: 15px;
   font-size: $paragraph;
-  overflow: scroll;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
   max-height: 15vh;
 }
 
 .instagram{
   height: 60vh;
 }
 
-@media screen and (min-width: 600px){
+@media screen and (min-width: 641px){
```

Comment:
where did you come up with these breakpoints?
These should also be Sass variables so they stay consistent across the site.